src/routes/safe/components/Apps/confirmTransactions.js

```diff
@@ -38,9 +48,31 @@ const confirmTransactions = (
   closeModal: () => void,
   onConfirm: () => void,
 ) => {
+  const isTxError = txs.some((t) => {
+    return (
+      !t.to ||
+      typeof t.to !== 'string' ||
```

Comment:
`t.to` should be a valid ethereum address and not just a string, shouldnt it?

Reply:
done!